### DISCUSSION TITLE
Follow User with Private Account

### DISCUSSION THREAD

**Discussion by jlosito** [community/community#41130] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am trying to follow another user whose account is marked as private. Due to their account being private, their follow button is removed from their profile. Is there no way to follow this person now? Note that when I try to mark my profile as being private as an experiment, I'm still able to see a list of people who are following me, so I'm assuming this capability is not lost entirely.

**Comment by Wojtekb30** [comment, marked as answer]
From what I know (and my own tests) it is not possible to follow such private account anymore. But if the user got followers when their account was public, then the followers remain.

**Reply by Wojtekb30** [reply]
By the way in this case you can ask the person to make their account public for a short while so you can follow them.
